### PR TITLE
Add blsfuzz command

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
TODO

- [ ] Incorporate existing corpus
- [x] Smarter Fuzzing algorithm
- [x] Test with geth and open ethereum's evm

### CODE REVIEW

File: fuzzing/bls12381.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+			To:         dest.Hex(),
+			PrivateKey: hexutil.MustDecode("0x45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"),
+		}
+		gst.SetTx(tx)
+	}
+	return gst
+}
+
+func RandCallBLS() []byte {
+	p := program.NewProgram()
```
Comment (holiman) [inline fuzzing/bls12381.go:70]: One thing you _could_ do, is make this shim contract call each one of the precomples, serially, and store their output in slots [0 .. 9].

Comment (MariusVanDerWijden) [reply fuzzing/bls12381.go:70]: I tried, I'm not sure about my offset calculation, PTAL

**Comment by holiman** [comment]
You have updated the go-ethereum module to a more recent version, which is a very good change. Perhaps we can merge this, even if everything doesn't properly work, to at least get that change in? I think some other stuff need to be fixed after that update too, but I can do that in a separate PR

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.jsonl
```
Comment (holiman) [inline .gitignore:1]: We actually have a few `jsonl` files in git, which are _supposed_ to be tracked. I'm not sure how that plays with this entry here?

Comment (MariusVanDerWijden) [reply .gitignore]: I moved the .gitignore into the `cmd` subfolder, so it won't interfere with our testdata but will ignore output generated by the binaries

**Pull request merged**